src/test/groovy/graphql/incremental/IncrementalExecutionResultTest.groovy

```diff
@@ -40,13 +40,15 @@ class IncrementalExecutionResultTest extends Specification {
                 ])
                 .hasNext(true)
                 .incremental([defer1, stream1, stream2])
+                .extensions([some: "map"])
```

Comment:
this is a similar `extensions` function for another class that I've added as well